protocol/daemons/flags/flags.go

```diff
@@ -74,7 +81,9 @@ func GetDefaultDaemonFlags() DaemonFlags {
 	if defaultDaemonFlags == nil {
 		defaultDaemonFlags = &DaemonFlags{
 			Shared: SharedFlags{
-				SocketAddress: "/tmp/daemons.sock",
+				SocketAddress:                 "/tmp/daemons.sock",
+				DisablePanicOnDaemonFailure:   false,
```

Comment:
I think we use `Enabled` (as opposed to disabled) throughout our codebase. For consistency, can we use this format?

```suggestion
				PanicOnDaemonFailureEnabled:   true,
```